applications/nrf5340_audio/src/utils/error_handler.c

```diff
@@ -8,48 +8,43 @@
 #include <zephyr/sys/reboot.h>
 #include <zephyr/fatal.h>
 #include <zephyr/logging/log_ctrl.h>
-
-#include "board.h"
-#include "led.h"
+#include <zephyr/drivers/gpio.h>
 
 /* Print everything from the error handler */
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(error_handler, CONFIG_ERROR_HANDLER_LOG_LEVEL);
 
-static void led_error_indication(void)
-{
-#if defined(CONFIG_BOARD_NRF5340_AUDIO_DK_NRF5340_CPUAPP)
-	(void)led_on(LED_APP_RGB, LED_COLOR_RED);
-#elif defined(CONFIG_BOARD_NRF5340_AUDIO_DK_NRF5340_CPUNET)
-	(void)led_on(LED_NET_RGB, LED_COLOR_RED);
-#endif /* defined(CONFIG_BOARD_NRF5340_AUDIO_DK_NRF5340_CPUAPP) */
-}
+/* nRF5340 Audio DK center RGB LED */
+static const struct gpio_dt_spec center_led_r = GPIO_DT_SPEC_GET(DT_NODELABEL(rgb1_red), gpios);
+static const struct gpio_dt_spec center_led_g = GPIO_DT_SPEC_GET(DT_NODELABEL(rgb1_green), gpios);
+static const struct gpio_dt_spec center_led_b = GPIO_DT_SPEC_GET(DT_NODELABEL(rgb1_blue), gpios);
```

Comment:
This is now tied very heavily to our board. If the pin/pins were passed in with an init function this file could reside in NCS instead.

Reply:
That is possible, though, the system may fail before we have a chance to init the error handler.